app/views/schools/dashboard/show.html.erb

```diff
@@ -14,11 +14,11 @@
       <% if school_cohort.core_induction_programme? || school_cohort.full_induction_programme? %>
         <% if school_cohort.active_ecf_participant_profiles.any? %>
           <p class="govuk-body govuk-!-margin-bottom-5">
-            <%= govuk_link_to 'View your early career teacher and mentor details', schools_participants_path(cohort_id: school_cohort.cohort.start_year), class: "govuk-link--no-visited-state" %>
+            <%= govuk_link_to 'View your early career teacher and mentor details', roles_schools_cohort_path(cohort_id: school_cohort.cohort.start_year), class: "govuk-link--no-visited-state" %>
```

Comment:
I thought we were only showing this roles page before they had added participants. Has that changed?